src/tests/fixtures_lakify_recursive_json_nest/basic.expected.json

```diff
@@ -0,0 +1,45 @@
+{
+  "iati-activity": [
+    {
+      "@last-updated-datetime": "2014-09-10T07:15:37Z",
+      "iati-identifier": [
+        {
+          "text()": "ACT-1"
+        }
+      ],
+      "reporting-org": [
+        {
+          "@ref": "AA-AAA-123456789",
+          "@secondary-reporter": "0",
+          "@type": "40",
+          "narrative": [
+            {
+              "text()": "Organisation name"
+            },
+            {
+              "@{http://www.w3.org/XML/1998/namespace}lang": "fr",
```

Comment:
The namespace serialization is a bit longer-winded than I expected, but otherwise this is a good unit test.